_posts/2021-02-01-launching-the-solidity-forum.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+published: true
+title: 'Launching the Solidity Forum 🗃️'
+date: '2021-02-01'
+author: Franziska Heintel
+category: Announcements
+---
+
+In our effort to foster exchange of information, encourage more developers to give feedback about Solidity and join the discussions on language design and future direction of the compiler, we are happy to launch the [Solidity forum](https://forum.soliditylang.org/) today! 
+
+Moving forward the Solidity forum will be the dedicated place to discuss topics and questions related to…
+
+- The implementation and design of Solidity programming language.
+- The Solidity compiler.
+- Useful Solidity tips and code snippets.
+- Solidity documentation and its translation.
+- Discussions and announcements about Solidity releases.
+
+It will **not** be the place for…
+
+- Ad-hoc support questions. For urgent Solidity support questions, please use the [Solidity Gitter chat](https://gitter.im/ethereum/solidity).
```

Comment:
Maybe this should be a bit more clarified. The "Code Wizards" section below contradicts this a bit.

Also what about ethereum.stackexchange.com?

Reply:
true, I will make it clearer it's more for code snippets sharing and not an ad-hoc help channel